--- a/net/minecraft/client/Minecraft.java
+++ b/net/minecraft/client/Minecraft.java
@@ -424,6 +_,7 @@
       this.f_91066_ = new Options(this, this.f_91069_);
       this.f_91005_ = new Tutorial(this, this.f_91066_);
       this.f_91000_ = new HotbarManager(this.f_91069_, this.f_90988_);
+      if(false)
       f_90982_.info("Backend library: {}", (Object)RenderSystem.m_69517_());
       DisplayData displaydata;
       if (this.f_91066_.f_92129_ > 0 && this.f_91066_.f_92128_ > 0) {
@@ -432,7 +_,10 @@
          displaydata = p_91084_.f_101906_;
       }
 
+      if(false)
       Util.f_137440_ = RenderSystem.m_69576_();
+      f_91079_ = true;
+      if(false) {
       this.f_90989_ = new VirtualScreen(this);
       this.f_90990_ = this.f_90989_.m_110872_(displaydata, this.f_91066_.f_92123_, this.m_91270_());
       this.m_7440_(true);
@@ -455,11 +_,13 @@
       this.f_91042_ = new MainTarget(this.f_90990_.m_85441_(), this.f_90990_.m_85442_());
       this.f_91042_.m_83931_(0.0F, 0.0F, 0.0F, 0.0F);
       this.f_91042_.m_83954_(f_91002_);
+      }
       this.f_91036_ = new ReloadableResourceManager(PackType.CLIENT_RESOURCES);
       this.f_91038_.m_10506_();
       this.f_91066_.m_92145_(this.f_91038_);
       this.f_91039_ = new LanguageManager(this.f_91066_.f_92075_);
       this.f_91036_.m_7217_(this.f_91039_);
+      if(false) {
       this.f_90987_ = new TextureManager(this.f_91036_);
       this.f_91036_.m_7217_(this.f_90987_);
       this.f_91050_ = new SkinManager(this.f_90987_, new File(file1, "skins"), this.f_91048_);
@@ -494,15 +_,21 @@
       this.f_90996_ = new ItemInHandRenderer(this);
       this.f_91036_.m_7217_(this.f_90995_);
       this.f_90993_ = new RenderBuffers();
+      }
       this.f_91063_ = new GameRenderer(this, this.f_91036_, this.f_90993_);
+      if(false)
       this.f_91036_.m_7217_(this.f_91063_);
       this.f_91006_ = new PlayerSocialManager(this, this.f_193584_);
+      if(false) {
+      BlockEntityWithoutLevelRenderer blockentitywithoutlevelrenderer = null;
       this.f_91052_ = new BlockRenderDispatcher(this.f_91051_.m_119430_(), blockentitywithoutlevelrenderer, this.f_91040_);
       this.f_91036_.m_7217_(this.f_91052_);
       this.f_91060_ = new LevelRenderer(this, this.f_90993_);
       this.f_91036_.m_7217_(this.f_91060_);
+      }
       this.m_91271_();
       this.f_91036_.m_7217_(this.f_90997_);
+      if(false) {
       this.f_91061_ = new ParticleEngine(this.f_91073_, this.f_90987_);
       this.f_91036_.m_7217_(this.f_91061_);
       this.f_91053_ = new PaintingTextureManager(this.f_90987_);
@@ -547,6 +_,7 @@
             this.f_167847_.m_168556_();
          });
       }, false));
+      }
       if (s != null) {
          ConnectScreen.m_169267_(new TitleScreen(), this, new ServerAddress(s, i), (ServerData)null);
       } else {
@@ -894,6 +_,7 @@
       }
 
       this.f_91080_ = p_91153_;
+      if(false) {
       BufferUploader.m_166835_();
       if (p_91153_ != null) {
          this.f_91067_.m_91602_();
@@ -906,6 +_,7 @@
       }
 
       this.m_91341_();
+      }
    }
 
    public void m_91150_(@Nullable Overlay p_91151_) {
@@ -967,15 +_,17 @@
          this.f_90989_.close();
          this.f_90990_.close();
       }
-
    }
 
    private void m_91383_(boolean p_91384_) {
+	  if(false)
       this.f_90990_.m_85403_("Pre render");
       long i = Util.m_137569_();
+      if(false) {
       if (this.f_90990_.m_85411_()) {
          this.m_91395_();
       }
+      }
 
       if (this.f_91024_ != null && !(this.f_91081_ instanceof LoadingOverlay)) {
          CompletableFuture<Void> completablefuture = this.f_91024_;
@@ -1005,6 +_,7 @@
          this.f_91026_.m_7238_();
       }
 
+      if(false) {
       this.f_91067_.m_91523_();
       this.f_90990_.m_85403_("Render");
       this.f_91026_.m_6180_("sound");
@@ -1050,9 +_,11 @@
          RenderSystem.m_69830_(i1);
       }
 
+      }
       this.f_91026_.m_6182_("yield");
       Thread.yield();
       this.f_91026_.m_7238_();
+      if(false) {
       this.f_90990_.m_85403_("Post render");
       ++this.f_91016_;
       boolean flag = this.m_91091_() && (this.f_91080_ != null && this.f_91080_.m_7043_() || this.f_91081_ != null && this.f_91081_.m_7859_()) && !this.f_91007_.m_6992_();
@@ -1079,6 +_,8 @@
       }
 
       this.f_91026_.m_7238_();
+
+      }
    }
 
    private boolean m_91274_() {
@@ -1562,13 +_,16 @@
    }
 
    public void m_91398_() {
+	  net.hycrafthd.headless_minecraft.event_system.EventHooks.tick();
       if (this.f_91011_ > 0) {
          --this.f_91011_;
       }
 
+      if(false) {
       this.f_91026_.m_6180_("gui");
       this.f_91065_.m_193832_(this.f_91012_);
       this.f_91026_.m_7238_();
+      }
       this.f_91063_.m_109087_(1.0F);
       this.f_91005_.m_120578_(this.f_91073_, this.f_91077_);
       this.f_91026_.m_6180_("gameMode");
@@ -1576,10 +_,12 @@
          this.f_91072_.m_105287_();
       }
 
+      if(false) {
       this.f_91026_.m_6182_("textures");
       if (this.f_91073_ != null) {
          this.f_90987_.m_7673_();
       }
+      }
 
       if (this.f_91080_ == null && this.f_91074_ != null) {
          if (this.f_91074_.m_21224_() && !(this.f_91080_ instanceof DeathScreen)) {
@@ -1601,6 +_,7 @@
          this.f_91078_ = 10000;
       }
 
+      if(false) {
       if (this.f_91080_ != null) {
          Screen.m_96579_(() -> {
             this.f_91080_.m_96624_();
@@ -1610,16 +_,21 @@
       if (!this.f_91066_.f_92063_) {
          this.f_91065_.m_93091_();
       }
+      }
 
       if (this.f_91081_ == null && (this.f_91080_ == null || this.f_91080_.f_96546_)) {
+    	 if(false) {
          this.f_91026_.m_6182_("Keybindings");
          this.m_91279_();
+    	 }
+    	 m_91386_(net.hycrafthd.headless_minecraft.Main.left_click);
          if (this.f_91078_ > 0) {
             --this.f_91078_;
          }
       }
 
       if (this.f_91073_ != null) {
+    	 if(false) {
          this.f_91026_.m_6182_("gameRenderer");
          if (!this.f_91012_) {
             this.f_91063_.m_109148_();
@@ -1629,6 +_,7 @@
          if (!this.f_91012_) {
             this.f_91060_.m_109823_();
          }
+    	 }
 
          this.f_91026_.m_6182_("level");
          if (!this.f_91012_) {
@@ -1638,15 +_,17 @@
 
             this.f_91073_.m_104804_();
          }
-      } else if (this.f_91063_.m_109149_() != null) {
+      } else if (false && this.f_91063_.m_109149_() != null) {
          this.f_91063_.m_109086_();
       }
 
+      if(false) {
       if (!this.f_91012_) {
          this.f_91044_.m_120183_();
       }
 
       this.f_91043_.m_120389_(this.f_91012_);
+      }
       if (this.f_91073_ != null) {
          if (!this.f_91012_) {
             if (!this.f_91066_.f_92031_ && this.m_91278_()) {
@@ -1677,6 +_,7 @@
             }
          }
 
+         if(false) {
          this.f_91026_.m_6182_("animateTick");
          if (!this.f_91012_ && this.f_91073_ != null) {
             this.f_91073_.m_104784_(this.f_91074_.m_146903_(), this.f_91074_.m_146904_(), this.f_91074_.m_146907_());
@@ -1686,14 +_,17 @@
          if (!this.f_91012_) {
             this.f_91061_.m_107388_();
          }
+         }
       } else if (this.f_91009_ != null) {
          this.f_91026_.m_6182_("pendingConnection");
          this.f_91009_.m_129483_();
       }
 
+      if(false) {
       this.f_91026_.m_6182_("keyboard");
       this.f_91068_.m_90931_();
       this.f_91026_.m_7238_();
+      }
    }
 
    private boolean m_91278_() {
@@ -2030,6 +_,7 @@
       progressscreen.m_6309_(new TranslatableComponent("connect.joining"));
       this.m_91362_(progressscreen);
       this.f_91073_ = p_91157_;
+      if(false)
       this.m_91324_(p_91157_);
       if (!this.f_91010_) {
          AuthenticationService authenticationservice = new YggdrasilAuthenticationService(this.f_91030_);
@@ -2057,6 +_,7 @@
       this.f_91006_.m_194060_();
       IntegratedServer integratedserver = this.f_91007_;
       this.f_91007_ = null;
+      if(false)
       this.f_91063_.m_109150_();
       this.f_91072_ = null;
       NarratorChatListener.f_93311_.m_93328_();
@@ -2073,6 +_,7 @@
          }
 
          this.f_91037_.m_118586_();
+         if(false)
          this.f_91065_.m_93089_();
          this.f_91008_ = null;
          this.f_91010_ = false;
@@ -2080,6 +_,7 @@
       }
 
       this.f_91073_ = null;
+      if(false)
       this.m_91324_((ClientLevel)null);
       this.f_91074_ = null;
       SkullBlockEntity.m_196704_();
@@ -2087,6 +_,7 @@
 
    private void m_91362_(Screen p_91363_) {
       this.f_91026_.m_6180_("forcedTick");
+      if(false)
       this.f_91043_.m_120405_();
       this.f_91075_ = null;
       this.f_91009_ = null;
@@ -2266,6 +_,7 @@
       p_167851_.m_143522_("Launched Version", () -> {
          return p_167854_;
       });
+      if(false) {
       p_167851_.m_143522_("Backend library", RenderSystem::m_69517_);
       p_167851_.m_143522_("Backend API", RenderSystem::m_69516_);
       p_167851_.m_143522_("Window size", () -> {
@@ -2316,6 +_,7 @@
       }
 
       p_167851_.m_143522_("CPU", GlUtil::m_84819_);
+      }
       return p_167851_;
    }
 
