--- a/net/minecraft/client/multiplayer/ClientLevel.java
+++ b/net/minecraft/client/multiplayer/ClientLevel.java
@@ -493,14 +_,17 @@
    }
 
    public void m_7260_(BlockPos p_104685_, BlockState p_104686_, BlockState p_104687_, int p_104688_) {
+	  if(false)
       this.f_104562_.m_109544_(this, p_104685_, p_104686_, p_104687_, p_104688_);
    }
 
    public void m_6550_(BlockPos p_104759_, BlockState p_104760_, BlockState p_104761_) {
+	  if(false) 
       this.f_104562_.m_109721_(p_104759_, p_104760_, p_104761_);
    }
 
    public void m_104793_(int p_104794_, int p_104795_, int p_104796_) {
+	  if(false)
       this.f_104562_.m_109490_(p_104794_, p_104795_, p_104796_);
    }
 
@@ -513,15 +_,18 @@
    }
 
    public void m_6801_(int p_104634_, BlockPos p_104635_, int p_104636_) {
+	  if(false)
       this.f_104562_.m_109774_(p_104634_, p_104635_, p_104636_);
    }
 
    public void m_6798_(int p_104743_, BlockPos p_104744_, int p_104745_) {
+	  if(false)
       this.f_104562_.m_109506_(p_104743_, p_104744_, p_104745_);
    }
 
    public void m_5898_(@Nullable Player p_104654_, int p_104655_, BlockPos p_104656_, int p_104657_) {
       try {
+    	 if(false)
          this.f_104562_.m_109532_(p_104654_, p_104655_, p_104656_, p_104657_);
       } catch (Throwable throwable) {
          CrashReport crashreport = CrashReport.m_127521_(throwable, "Playing level event");
@@ -535,18 +_,22 @@
    }
 
    public void m_7106_(ParticleOptions p_104706_, double p_104707_, double p_104708_, double p_104709_, double p_104710_, double p_104711_, double p_104712_) {
+	  if(false)
       this.f_104562_.m_109743_(p_104706_, p_104706_.m_6012_().m_123742_(), p_104707_, p_104708_, p_104709_, p_104710_, p_104711_, p_104712_);
    }
 
    public void m_6493_(ParticleOptions p_104714_, boolean p_104715_, double p_104716_, double p_104717_, double p_104718_, double p_104719_, double p_104720_, double p_104721_) {
-      this.f_104562_.m_109743_(p_104714_, p_104714_.m_6012_().m_123742_() || p_104715_, p_104716_, p_104717_, p_104718_, p_104719_, p_104720_, p_104721_);
+	  if(false)
+	  this.f_104562_.m_109743_(p_104714_, p_104714_.m_6012_().m_123742_() || p_104715_, p_104716_, p_104717_, p_104718_, p_104719_, p_104720_, p_104721_);
    }
 
    public void m_7107_(ParticleOptions p_104766_, double p_104767_, double p_104768_, double p_104769_, double p_104770_, double p_104771_, double p_104772_) {
-      this.f_104562_.m_109752_(p_104766_, false, true, p_104767_, p_104768_, p_104769_, p_104770_, p_104771_, p_104772_);
+	  if(false)
+	  this.f_104562_.m_109752_(p_104766_, false, true, p_104767_, p_104768_, p_104769_, p_104770_, p_104771_, p_104772_);
    }
 
    public void m_6485_(ParticleOptions p_104774_, boolean p_104775_, double p_104776_, double p_104777_, double p_104778_, double p_104779_, double p_104780_, double p_104781_) {
+	  if(false)
       this.f_104562_.m_109752_(p_104774_, p_104774_.m_6012_().m_123742_() || p_104775_, true, p_104776_, p_104777_, p_104778_, p_104779_, p_104780_, p_104781_);
    }
 
