--- a/net/minecraft/client/renderer/GameRenderer.java
+++ b/net/minecraft/client/renderer/GameRenderer.java
@@ -96,7 +_,7 @@
    private boolean f_182638_;
    private long f_109073_ = Util.m_137550_();
    protected LightTexture f_109074_;
-   private final OverlayTexture f_109075_ = new OverlayTexture();
+   private final OverlayTexture f_109075_ = null;
    private boolean f_109076_;
    private float f_109077_ = 1.0F;
    private float f_109078_;
@@ -229,8 +_,10 @@
       this.f_109059_ = p_109083_;
       this.f_109060_ = p_109084_;
       this.f_109055_ = p_109083_.m_91292_();
+      if(false) {
       this.f_109063_ = new MapRenderer(p_109083_.m_91097_());
       this.f_109074_ = new LightTexture(this, p_109083_);
+      }
       this.f_109064_ = p_109085_;
       this.f_109050_ = null;
    }
