--- a/net/minecraft/world/ticks/ScheduledTick.java
+++ b/net/minecraft/world/ticks/ScheduledTick.java
@@ -36,7 +_,7 @@
    };
 
    public ScheduledTick(T p_193383_, BlockPos p_193384_, long p_193385_, long p_193386_) {
-      this(p_193383_, p_193384_, p_193385_, TickPriority.NORMAL, p_193386_);
+      this((T)p_193383_, p_193384_, p_193385_, TickPriority.NORMAL, p_193386_);
    }
 
    public ScheduledTick(T f_193376_, BlockPos f_193377_, long f_193378_, TickPriority f_193379_, long f_193380_) {
